### ISSUE TITLE
Missing asset-manifest meta

### ISSUE THREAD

**Issue by saverio-kantox** [ember-cli-deploy/ember-cli-deploy-json-config#30]
The generated index.html in build has a 
```html
<meta name="my-app/config/asset-manifest" content="%7B%22bundles%22%3A%7B%7D%7D" />
```
tag that is not transferred to the `index.json` file.

**Comment by lukemelia** [comment]
@saverio-kantox is this being added by an addon that uses `ember-asset-loader`?

**Comment by achambers** [comment]
@saverio-kantox This is because the default selector for the `meta` tag doesn't match on that meta tag. eg

```json
meta: {
  selector: 'meta[name*="/config/environment"]',
  attributes: ['name', 'content']
}
```

You will need to specify the `jsonBlueprint` [config property](https://github.com/ember-cli-deploy/ember-cli-deploy-json-config#jsonblueprint) and update that selector to also include your meta tag.

You can do this in a similar way to the example [HERE](https://github.com/ember-cli-deploy/ember-cli-deploy-json-config#includecontent) where you can just update the existing default config with the correct selector that matches both meta tags.

**Comment by saverio-kantox** [comment]
Indeed, I can configure it, but `ember-asset-loader` and `ember-engines` (that depends on the former) are becoming part of the standard, so it would be reasonable to have this meta added by default.
